static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -6,18 +6,36 @@
     padding: 0 $gs-gutter / 4 $gs-baseline $gs-gutter / 4;
 }
 
+.contributions__epic {
+    border-top: 1px solid $multimedia-main-2;
+    background-color: #efefec;
+    padding: 0 $gs-gutter / 4 $gs-baseline $gs-gutter / 4;
+}
+
 .contributions__title {
     @include fs-header(2);
     margin: 0;
 }
 
+.contributions__title-epic {
+    margin-bottom: $gs-baseline;
+}
+
 .contributions__paragraph  {
     font-size: 1em;
     line-height: 20px;
     font-family: $f-serif-headline;
     margin-bottom: $gs-baseline;
 }
 
+.contributions__paragraph-epic {
+    line-height: 1.5;
+    font-family: $f-serif-text;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
Shouldn't need any font-smoothing in here. Also, we should be using the mixins - did you work out what was different with the one yesterday?